draft-fossati-tls-attestation.md

```diff
@@ -848,10 +848,11 @@ in the specular extension and proceed by invoking a local API
 attestation using the nonce supplied by the verifier.  The returned
 evidence binds the identity key (TIK-S) with the platform identity and
 security state, packaged into a CAB.  The server then signs a transcript
```

Comment:
Is it really the server that actually does the signature? It does not seem to be consistent with [Fig. 12](https://tls-attestation.github.io/draft-tls-attestation/draft-fossati-tls-attestation.html#figure-12).

Reply:
It is. I guess, Fig 12 is a bit misleading, we'll have to fix that, but the attestation service is part of the server there.